db_stress_tool/expected_state.cc

```diff
@@ -144,29 +147,44 @@ Status FileExpectedState::Open(bool create) {
 
   Status status;
   if (create) {
-    std::unique_ptr<WritableFile> wfile;
-    const EnvOptions soptions;
-    status = default_env->NewWritableFile(expected_state_file_path_, &wfile,
-                                          soptions);
+    status = CreateFile(default_env, EnvOptions(), expected_state_file_path_,
```

Comment:
I wonder if it may be cleaner to use early returns on `if (!status.ok())` instead to avoid the repeated nesting of `if (status.ok())`. Either way does not seem great

Reply:
Fixed